cvs diff: Diffing svx
cvs diff: Diffing svx/doc
cvs diff: Diffing svx/doc/UML
cvs diff: Diffing svx/inc
cvs diff: Diffing svx/inc/pch
cvs diff: Diffing svx/inc/svx
cvs diff: Diffing svx/inc/svx/sdr
cvs diff: Diffing svx/inc/svx/sdr/animation
cvs diff: Diffing svx/inc/svx/sdr/contact
cvs diff: Diffing svx/inc/svx/sdr/event
cvs diff: Diffing svx/inc/svx/sdr/mixer
cvs diff: Diffing svx/inc/svx/sdr/overlay
cvs diff: Diffing svx/inc/svx/sdr/properties
cvs diff: Diffing svx/inc/svx/sdr/table
cvs diff: Diffing svx/prj
Index: svx/prj/build.lst
===================================================================
RCS file: /cvs/graphics/svx/prj/build.lst,v
retrieving revision 1.28
retrieving revision 1.28.18.1
diff -p -u -u -p -b -w -B -r1.28 -r1.28.18.1
--- svx/prj/build.lst	12 Mar 2008 09:34:44 -0000	1.28
+++ svx/prj/build.lst	28 Mar 2008 11:19:36 -0000	1.28.18.1
@@ -1,6 +1,7 @@
 sx	svx	:	SUN:officenames connectivity xmloff linguistic jvmfwk avmedia configmgr ucb fileaccess NULL
 sx	svx										usr1	-	all	sx_mkout NULL
 sx	svx\inc									nmake	-	all	sx_inc NULL
+sx	svx\uiconfig\layout									nmake	-	all	sx_layout NULL
 sx	svx\prj									get		-	all	sx_prj NULL
 sx	svx\win\res								get		-	all	sx_wres NULL
 sx	svx\os2\res								get		-	all	sx_ores NULL
Index: svx/prj/d.lst
===================================================================
RCS file: /cvs/graphics/svx/prj/d.lst,v
retrieving revision 1.136
retrieving revision 1.136.18.2
diff -p -u -u -p -b -w -B -r1.136 -r1.136.18.2
--- svx/prj/d.lst	12 Mar 2008 09:35:06 -0000	1.136
+++ svx/prj/d.lst	31 Mar 2008 14:52:10 -0000	1.136.18.2
@@ -656,3 +656,5 @@ mkdir: %_DEST%\inc%_EXT%\svx\sdr\table
 ..\inc\svx\svdtext.hxx %_DEST%\inc%_EXT%\svx\svdtext.hxx
 ..\inc\svx\svdotable.hxx %_DEST%\inc%_EXT%\svx\svdotable.hxx
 ..\inc\svx\selectioncontroller.hxx %_DEST%\inc%_EXT%\svx\selectioncontroller.hxx
+
+..\%__SRC%\bin\*-layout.zip %_DEST%\pck%_EXT%\*.*
cvs diff: Diffing svx/qa
cvs diff: Diffing svx/qa/unoapi
cvs diff: Diffing svx/qa/unoapi/testdocuments
cvs diff: Diffing svx/sdi
cvs diff: Diffing svx/source
cvs diff: Diffing svx/source/accessibility
cvs diff: Diffing svx/source/customshapes
cvs diff: Diffing svx/source/dialog
cvs diff: Diffing svx/source/editeng
cvs diff: Diffing svx/source/engine3d
cvs diff: Diffing svx/source/fmcomp
cvs diff: Diffing svx/source/form
cvs diff: Diffing svx/source/gallery2
cvs diff: Diffing svx/source/gengal
cvs diff: Diffing svx/source/inc
cvs diff: Diffing svx/source/init
cvs diff: Diffing svx/source/intro
cvs diff: Diffing svx/source/items
cvs diff: Diffing svx/source/mnuctrls
cvs diff: Diffing svx/source/msfilter
cvs diff: Diffing svx/source/options
cvs diff: Diffing svx/source/outliner
cvs diff: Diffing svx/source/sdr
cvs diff: Diffing svx/source/sdr/animation
cvs diff: Diffing svx/source/sdr/contact
cvs diff: Diffing svx/source/sdr/event
cvs diff: Diffing svx/source/sdr/mixer
cvs diff: Diffing svx/source/sdr/overlay
cvs diff: Diffing svx/source/sdr/properties
cvs diff: Diffing svx/source/smarttags
cvs diff: Diffing svx/source/src
cvs diff: Diffing svx/source/stbctrls
cvs diff: Diffing svx/source/svdraw
cvs diff: Diffing svx/source/svrtf
cvs diff: Diffing svx/source/svxlink
cvs diff: Diffing svx/source/table
cvs diff: Diffing svx/source/tbxctrls
cvs diff: Diffing svx/source/toolbars
cvs diff: Diffing svx/source/unodialogs
cvs diff: Diffing svx/source/unodialogs/inc
cvs diff: Diffing svx/source/unodialogs/textconversiondlgs
cvs diff: Diffing svx/source/unodraw
cvs diff: Diffing svx/source/unoedit
cvs diff: Diffing svx/source/unogallery
cvs diff: Diffing svx/source/xml
cvs diff: Diffing svx/source/xoutdev
cvs diff: Diffing svx/uiconfig
cvs diff: Diffing svx/uiconfig/accelerator
cvs diff: Diffing svx/uiconfig/accelerator/en-US
cvs diff: Diffing svx/uiconfig/accelerator/es
cvs diff: Diffing svx/uiconfig/layout
Index: svx/uiconfig/layout/delzip
===================================================================
RCS file: svx/uiconfig/layout/delzip
diff -N svx/uiconfig/layout/delzip
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ svx/uiconfig/layout/delzip	3 Apr 2008 08:39:33 -0000	1.1.2.2
@@ -0,0 +1 @@
+empty
\ No newline at end of file
Index: svx/uiconfig/layout/layout.mk
===================================================================
RCS file: svx/uiconfig/layout/layout.mk
diff -N svx/uiconfig/layout/layout.mk
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ svx/uiconfig/layout/layout.mk	31 Mar 2008 15:01:53 -0000	1.1.2.1
@@ -0,0 +1,33 @@
+# TODO: move to solenv/inc
+# copies: sw/uiconfig/layout svx/uiconfig/layout
+
+TRALAY=tralay
+XML_DEST=$(DLLDEST)
+XML_LANGS=$(alllangiso)
+
+ALL_XMLS=$(foreach,i,$(XML_FILES) $(XML_DEST)/$i) $(foreach,i,$(XML_LANGS) $(foreach,j,$(XML_FILES) $(XML_DEST)/$i/$j))
+
+# Must remove the -j (no duplicate base file names) flag
+ZIPUPDATE=-u
+XML_ZIP = $(PRJNAME)-layout
+
+ALLTAR: $(XML_ZIP)
+
+$(XML_ZIP): $(ALL_XMLS)
+
+ZIP1DIR=$(XML_DEST)
+ZIP1TARGET=$(XML_ZIP)
+ZIP1LIST=$(ALL_XMLS:s@$(XML_DEST)/@@)
+
+$(foreach,i,$(XML_LANGS) $(XML_DEST)/$i/%.xml): %.xml
+	-$(MKDIR) $(@:d)
+	@echo $(foreach,i,$(XML_LANGS) $(XML_DEST)/$i/%.xml): %.xml
+	$(TRALAY) -m localize.sdf -o $(XML_DEST) -l $(XML_LANGS:f:t" -l ") $<
+
+$(XML_DEST)/%.xml: %.xml
+	-$(MKDIR) $(@:d)
+	$(COPY) $< $@
+
+# Don't want to overwrite filled localize.sdf with empty template
+template.sdf:
+	$(foreach,i,$(XML_FILES) $(TRALAY) -l en-US $i) > $@
Index: svx/uiconfig/layout/localize.sdf
===================================================================
RCS file: svx/uiconfig/layout/localize.sdf
diff -N svx/uiconfig/layout/localize.sdf
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ svx/uiconfig/layout/localize.sdf	28 Mar 2008 11:19:36 -0000	1.1.2.1
@@ -0,0 +1,493 @@
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	en-US	Zoom & View Layout				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	en-US	Zoom factor				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	en-US	Whole Page				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_AUTOMATIC_label				0	en-US	~Automatic				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_SINGLE_label				0	en-US	~Single page				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_COLUMNS_label				0	en-US	~Columns				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	ED_COLUMNS_text				0	en-US	1				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	CHK_BOOK_label				0	en-US	~Book mode				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_VIEWLAYOUT_text				0	en-US	View layout				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_AUTOMATIC_label				0	de	~Automatisch				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_SINGLE_label				0	de	~Einzelne Seite				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_COLUMNS_label				0	de	~Spalten				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	ED_COLUMNS_text				0	de	1				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	CHK_BOOK_label				0	de	~Buch Modus				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_VIEWLAYOUT_text				0	de	Ansichtlayout				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	de	Maßstab & Ansichtlayout				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_AUTOMATIC_label				0	nl	~Automatisch				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_SINGLE_label				0	nl	~Enkele pagina				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_COLUMNS_label				0	nl	~Kolommen				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	ED_COLUMNS_text				0	nl	1				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	CHK_BOOK_label				0	nl	~Boek modus				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_VIEWLAYOUT_text				0	nl	Beeldopmaak				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	nl	Zoom & Beeldopmaak				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	en-US	Page Width				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	en-US	Optimal				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_200_label				0	en-US	200 %				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_150_label				0	en-US	15~0 %				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_100_label				0	en-US	100 %				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_75_label				0	en-US	75 %				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_50_label				0	en-US	50 %				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	en-US	Variable				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	ED_USER_text				0	en-US	100%				20080215 15:20:14
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	af	Zoemfaktor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ar	عامل تكبير/تصغير				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	as-IN	জুম কাৰক				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	be-BY	Маштаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	bg	Мащаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	br	Parenn				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	bs	Faktor povećanja				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ca	Factor de l'escala				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	cs	Faktor zvětšení				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	cy	Ffactor chwyddo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	da	Zoomfaktor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	de	Zoomfaktor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	dz	ཆ་རྐྱེན་རྒྱས་ཟུམ།				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	el	Συντελεστής εστίασης				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	en-GB	Zoom factor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	en-ZA	Zoom factor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	eo	Zoma faktoro				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	es	Escala				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	et	Suurendusaste				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	fa	عامل زوم				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	fi	Zoom-kerroin				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	fr	Facteur				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ga	Fachtóir súmála				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	gl	Factor de zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	gu	ઝુમનો અવયવ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	gu-IN	ઝુમનો અવયવ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	he	גורם זום				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	hi-IN	जूम कारक				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	hr	Faktor uvećanja				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	hu	Nagyítás mértéke				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	it	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ja	ズーム率				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ka	მაშტაბირების ფაქტორი				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	km	កត្តា​ពង្រីក				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ko	확대/축소				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ku	Rêjeya Nêzîkirinê				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	lt	Mastelio parinkimas				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	mk	Фактор на зумирање				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ml-IN	സൂം ഘടകം				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	mr-IN	झूम गुणक				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	nb	Skaleringsfaktor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ne	जुम खण्ड				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	nl	Zoomfactor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	nn	Skaleringsfaktor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	nr	Ifektha yokukhulisa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ns	Godiša ntlha				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	or-IN	ଜୁମ୍ ଫ୍ଯାକଟର				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	pa-IN	ਅਕਾਰ ਗੁਣਾਂਕ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	pl	Powiększenie				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	pt	Factor de zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	pt-BR	Fator de zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ru	Масштаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	rw	Igikubisho cy'ihindurangano				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	sh	Faktor uvećanja				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	sk	Faktor zväčšenia				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	sl	Faktor povečave				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	sr	Фактор увећања				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ss	Ifektha yaZumu				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	st	Zoom factore				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	sv	Zoomfaktor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	sw-TZ	Kiwango tutushi				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ta	Zoom factor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ta-IN	Zoom factor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	te-IN	జూమ్ కారకం				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	tg	Коэффитсиенти масштаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	th	Zoom factor				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	tn	Siana lebaka				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	tr	Yakınlaştırma oranı				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ts	Xivumbi xa ku kurisa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	uk	Коефіцієнт масштабування				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ur-IN	زوم  فیكٹر				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	uz	Kattalashtirish darajasi				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	ve	Fekhitha ya Zumu				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	vi	Hệ số thu phóng				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	xh	Umba wokwandisa unciphisa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	zh-CN	显示比例				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	zh-TW	顯示比例				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	FL_ZOOM_text				0	zu	Ingxenye yokusondeza				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	af	Zoem				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ar	تكبير/تصغير				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	as-IN	জুম				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	be-BY	Маштаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	bg	Мащаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	br	Zoum				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	bs	Uvećaj				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ca	Escala				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	cs	Lupa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	cy	Chwyddo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	da	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	dz	རྒྱས་ཟུམ།				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	el	Εστίαση				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	en-GB	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	en-ZA	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	eo	Zomi				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	es	Escala				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	et	Suurendus				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	fa	زوم				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	fi	Zoomaus				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	fr	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ga	Súmáil				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	gl	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	gu	નાનુમોટુ કરવુ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	gu-IN	નાનુમોટુ કરવુ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	he	זום				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	hi-IN	जूम				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	hr	Uvećaj/umanji				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	hu	Nagyítás				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	it	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ja	ズーム				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ka	მაშტაბი				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	km	ពង្រីក​				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ko	배율				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ku	Mezinkirin				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	lt	Mastelis				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	mk	Зумирање				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ml-IN	സൂം				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	mr-IN	झूम करा				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	nb	Skaler				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ne	जुम				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	nn	Skaler				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	nr	Khula msinyana				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ns	Godiša				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	or-IN	ଜୁମ୍				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	pa-IN	ਜ਼ੂਮ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	pl	Skala				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	pt	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	pt-BR	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ru	Масштаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	rw	Ihindurangano				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	sh	Uveličanje				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	sk	Lupa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	sl	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	sr	Увеличање				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ss	Zumu				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	st	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	sv	Skala				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	sw-TZ	Tutusha				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ta	கிட்டப்பார்				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ta-IN	கிட்டப்பார்				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	te-IN	జూమ్				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	tg	Масштаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	th	Zoom				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	tn	Siana				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	tr	Yakınlaştır				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ts	Ku kurisa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	uk	Масштаб				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ur-IN	 زوم				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	uz	Kattalashtirish				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	ve	Zumu				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	vi	Thu/phóng				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	xh	Ukwandisa unciphise				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	zh-CN	显示比例				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	zh-TW	顯示比例				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	RID_SVXDLG_ZOOM_title				0	zu	Sondeza				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	af	~Optimale				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ar	الأمثل~				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	be-BY	Аптымальна				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	bg	~Оптимален				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	br	~Gwellek				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	bs	~Optimalno				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ca	Ò~ptim				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	cs	Optimální				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	cy	~Gorau				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	da	~Optimal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	de	~Optimal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	dz	གང་དྲག(~O)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	el	Βέ~λτιστα				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	en-GB	~Optimal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	en-ZA	~Optimal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	eo	Optimuma				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	es	Ó~ptimo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	et	Optimaalne				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	fa	~بهینه				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	fi	O~ptimaalinen				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	fr	~Optimal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ga	~Optamach				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	gl	~Ideal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	gu	શ્રેષ્ઠ (~O)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	gu-IN	શ્રેષ્ઠ (~O)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	he	מיטבי				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	hi-IN	~Optimal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	hr	~Optimalno				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	hu	~Optimális				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	it	~Ottimale				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ja	最適(~O)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ka	~ოფტიმალური				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	km	ប្រសើរ​បំផុត 				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ko	최적(~O)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ku	D~înamik				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	lt	~Optimalus				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	mk	~Оптимално				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	nb	~Best mulig				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ne	इष्टतम				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	nl	~Optimaal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	nn	~Best mogleg				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	nr	~Ngaletha ipumelelo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ns	~Kgolo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	or-IN	ଅପଟିମାଲ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	pa-IN	ਅਨੁਕੂਲ(~O)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	pl	~Optymalny				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	pt	~Optimizado				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	pt-BR	~Ideal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ru	Оптимально				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	rw	Gihamye				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	sh	~Optimalan				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	sk	~Optimálny				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	sl	~Optimalno				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	sr	~Оптималан				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ss	~Kwekugcina				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	st	~E fetisisang				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	sv	Optimal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	sw-TZ	Sadifu				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ta	~சிறப்பு				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ta-IN	~சிறப்பு				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	te-IN	అనుకూలమైన				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	tg	Оптималӣ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	th	เ~หมาะที่สุด				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	tn	~Nepagetse				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	tr	En Uygun				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ts	~Mpimo wa le makumu				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	uk	Оптимальний				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ur-IN	مناسب				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	uz	~Optimal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	ve	~Oputimala				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	vi	Tối ư~u				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	xh	~Okufaneleke ncam				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	zh-CN	最佳(~O)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	zh-TW	最合適(~O)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_OPTIMAL_label				0	zu	~Okuthandwayo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	af	~Bladsybreedte				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ar	~عرض الصفحة				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	as-IN	পৃষ্ঠাৰ প্রস্থ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	be-BY	Шырыня старонкі				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	bg	~Ширина на страницата				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	br	~Led pajenn				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	bs	Širina ~stranice				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ca	~Ample de pàgina				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	cs	Šířka stránky				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	cy	~Lled tudalen				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	da	Side~bredde				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	de	Seiten~breite				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	dz	ཤོག་ལེབ་ཀྱི་རྒྱ་ཚད།(~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	el	Πλάτος σελίδας				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	en-GB	~Page Width				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	en-ZA	~Page Width				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	eo	Pagxa largxo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	es	~Ancho de página				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	et	Lehekülje laius				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	fa	~عرض صفحه				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	fi	~Sivun leveys				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	fr	~Largeur de page				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ga	Leithead ~Leathanaigh				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	gl	Largura de ~páxina				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	gu	પાનું પહોળાઈ(~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	gu-IN	પાનું પહોળાઈ(~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	he	רוחב עמוד				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	hi-IN	पृष्ठ चौड़ाई (~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	hr	~Širina stranice				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	hu	~Oldalszélesség				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	it	Larg~h. pagina				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ja	ページ幅(~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ka	~გვერდის სიგანე				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	km	ទទឹង​ទំព័រ 				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ko	페이지 너비(~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	lt	~Puslapio plotis				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	mk	Ширина на ~страница				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ml-IN	പേജിന്റെ വീതി				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	mr-IN	~पृष्ठाची रुंदी				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	nb	~Sidebredde				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ne	पृष्ठ चौडाइ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	nl	~Paginabreedte				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	nn	~Sidebreidd				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	nr	~Ububanzi bekhasi				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ns	~Bophara bja Letlakala				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	or-IN	ପୃଷ୍ଠାର ଓସାର				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	pa-IN	ਸਫ਼ਾ ਚੌੜਾਈ(~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	pl	Szerokość ~strony				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	pt	~Largura da página				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	pt-BR	Largura da ~página				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ru	По ширине страницы				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	rw	Ubugari bw'Urupapuro				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	sh	~Širina strane				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	sk	Šírka ~strany				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	sl	Širina s~trani				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	sr	~Ширина стране				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ss	~Bubanti belipheji				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	st	~Bophara ba Leqephe				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	sv	Sidbredd				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	sw-TZ	Upana Ukurasa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ta	பக்க அகலம்				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ta-IN	பக்க அகலம்				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	te-IN	పుటము యొక్క వెడెల్పు				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	tg	Ба бари саҳифа				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	th	ความกว้างของ~หน้า				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	tn	~Tsebe Bophara				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	tr	Sayfa Genişliği				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ts	~Ku anama ka pheji				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	uk	~Page Width				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ur-IN	صفحہ کی چوڑائی				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	uz	Betning ~eni				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	ve	~Vhuphara ha Siatari				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	vi	Độ ~rộng trang				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	xh	~Ububanzi Bekhasi				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	zh-CN	页面宽度(~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	zh-TW	頁面寬度(~P)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_PAGE_WIDTH_label				0	zu	Ububanzi be~khasi				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	af	~Veranderlike				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ar	م~تغير				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	as-IN	চলক				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	be-BY	Зменная				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	bg	~Променлив				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	br	~All				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	bs	~Promjenljivo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ca	~Variable				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	cs	Proměnný				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	cy	~Newidiol				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	da	~Variabel				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	de	~Stufenlos				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	dz	འགྱུར་ཅན།(~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	el	Προ~σαρμογή				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	en-GB	~Variable				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	en-ZA	~Variable				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	eo	Variablo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	es	~Variable				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	et	Kohandatud				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	fa	~متغیر				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	fi	~Vaihtuva				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	fr	~Autre				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ga	~Luaineach				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	gl	~Variable				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	gu	ચલ(~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	gu-IN	ચલ(~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	he	משתנה				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	hi-IN	चर (~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	hr	~Varijabilno				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	hu	~Egyéb:				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	it	~Variabile				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ja	任意(~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ka	~ცვლადი				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	km	អថេរ 				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ko	가변(~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ku	~Guherbar				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	lt	~Kitoks				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	mk	Променли~во				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ml-IN	വെരിയബിള്				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	mr-IN	~वेरीयेबल				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	nb	~Tilpass				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ne	चल				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	nl	~Variabel				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	nn	~Tilpass				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	nr	~Kwehlukano-gama				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ns	~Fetogago				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	or-IN	ପରିବର୍ତ୍ତନଶୀଳ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	pa-IN	ਮੁੱਲ(~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	pl	~Inne				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	pt	~Variável				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	pt-BR	~Variável				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ru	Произвольно				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	rw	Bihinduka				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	sh	~Promenljiv				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	sk	~Premenný				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	sl	~Spremenljivo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	sr	~Променљив				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ss	~Kugucugucuka				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	st	~Variebole				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	sv	~Steglös				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	sw-TZ	Kibadili				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ta	மாறுவது				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ta-IN	மாறுவது				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	te-IN	చరరాశి				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	tg	~Тағирёбанда				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	th	~ตัวแปร				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	tn	~Farologane				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	tr	Değişken				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ts	~Lexi cincacincaka				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	uk	Змінний				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ur-IN	تغیر				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	uz	Oʻz~garuvchan				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	ve	~Variebulu				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	vi	~Biến đổi				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	xh	~Iziguquguquki				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	zh-CN	无级缩放(~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	zh-TW	無級縮放(~V)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_USER_label				0	zu	~Okushintshayo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	af	~Hele bladsy				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ar	~صفحة كاملة				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	as-IN	সম্পূর্ণ পৃষ্ঠা				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	be-BY	Старонка цалкам				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	bg	~Цяла страница				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	br	Pajenn a-bezh				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	bs	~Cijela stranica				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ca	~Tota la pàgina				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	cs	Celá stránka				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	cy	~Tudalen Gyfan				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	da	H~ele siden				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	de	~Ganze Seite				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	dz	ཤོག་ལེབ་ཧྲིལ་བུ།(~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	el	Ολόκληρη σελίδα				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	en-GB	~Entire Page				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	en-ZA	~Entire Page				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	eo	Integra pagxo				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	es	~Toda la página				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	et	Terve lehekülg				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	fa	~سراسر صفحه				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	fi	~Koko sivu				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	fr	Page entière				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ga	L~eathanach ar fad				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	gl	Páxina completa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	gu	આખુ પાનું(~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	gu-IN	આખુ પાનું(~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	he	עמוד שלם				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	hi-IN	सम्पूर्ण पृष्ठ (~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	hr	~Cijela stranica				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	hu	~Teljes oldal				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	it	~Pagina intera				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ja	ページ全体(~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ka	~მთლიანი გვერდი				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	km	ទំព័រទាំងមូល 				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ko	전체 페이지(~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	lt	~Visas puslapis				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	mk	~Цела страница				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ml-IN	മുഴുവന് പേജ്				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	mr-IN	~संपूर्ण पृष्ठ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	nb	~Hele siden				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ne	सम्पूर्ण पृष्ठ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	nl	H~ele pagina				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	nn	~Heile sida				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	nr	~Ikhasi Loke				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ns	~Letlakala Ka moka				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	or-IN	ସମଗ୍ର ପୃଷ୍ଠା				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	pa-IN	ਮੁਕੰਮਲ ਸਫ਼ਾ(~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	pl	~Cała strona				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	pt	~Página inteira				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	pt-BR	Página int~eira				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ru	Вся страница				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	rw	Paji yose				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	sh	~Cela strana				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	sk	~Celá strana				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	sl	~Cela stran				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	sr	~Цела страна				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ss	~Lonkhe lipheji				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	st	~Leqephe kaofela				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	sv	Hela sidan				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	sw-TZ	Ukurasa Mzima				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ta	முழுப்பக்கம்				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ta-IN	முழுப்பக்கம்				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	te-IN	పుటము మొత్తము				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	tg	Ба тамомми саҳифа				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	th	~ทั้งหน้า				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	tn	~Yotlhe Tsebe				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	tr	Tüm Sayfa				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ts	~Pheji hinkwaro				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	uk	~Entire Page				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ur-IN	پوراصفحہ				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	uz	Bu~tun bet				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	ve	~Siatari Lothe				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	vi	~Toàn trang				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	xh	~Ikhasi Liphelele				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	zh-CN	整页(~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	zh-TW	整頁(~E)				2002-02-02 02:02:02
+layout	svx\uiconfig\layout\zoom.xml	0	layout	BTN_WHOLE_PAGE_label				0	zu	Ikhasi ~lonke				2002-02-02 02:02:02
Index: svx/uiconfig/layout/makefile.mk
===================================================================
RCS file: svx/uiconfig/layout/makefile.mk
diff -N svx/uiconfig/layout/makefile.mk
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ svx/uiconfig/layout/makefile.mk	3 Apr 2008 16:47:47 -0000	1.1.2.4
@@ -0,0 +1,54 @@
+#*************************************************************************
+#
+#   OpenOffice.org - a multi-platform office productivity suite
+#
+#   $RCSfile$
+#
+#   $Revision$
+#
+#   last change: $Author$ $Date$
+#
+#   The Contents of this file are made available subject to
+#   the terms of GNU Lesser General Public License Version 2.1.
+#
+#
+#     GNU Lesser General Public License Version 2.1
+#     =============================================
+#     Copyright 2005 by Sun Microsystems, Inc.
+#     901 San Antonio Road, Palo Alto, CA 94303, USA
+#
+#     This library is free software; you can redistribute it and/or
+#     modify it under the terms of the GNU Lesser General Public
+#     License version 2.1, as published by the Free Software Foundation.
+#
+#     This library is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#     Lesser General Public License for more details.
+#
+#     You should have received a copy of the GNU Lesser General Public
+#     License along with this library; if not, write to the Free Software
+#     Foundation, Inc., 51 Franklin Street, 5th Floor, Boston,
+#     MA  02110-1301  USA
+#
+#*************************************************************************
+
+PRJ=../..
+PRJNAME=svx
+TARGET=layout
+
+.INCLUDE : settings.mk
+
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+
+all: ALLTAR
+
+XML_FILES=zoom.xml
+
+.INCLUDE : layout.mk
+
+.ELSE # ENABLE_LAYOUT != TRUE
+all .PHONY:
+.ENDIF # ENABLE_LAYOUT != TRUE
+
+.INCLUDE :  target.mk
Index: svx/uiconfig/layout/zoom.xml
===================================================================
RCS file: svx/uiconfig/layout/zoom.xml
diff -N svx/uiconfig/layout/zoom.xml
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ svx/uiconfig/layout/zoom.xml	4 Apr 2008 09:14:29 -0000	1.1.2.2
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- This is a template.  i18n translation is not performed in-place;
+     i18n translated xml files are generated from this template by
+     transex3/layout/tralay.  !-->
+
+<modaldialog xmlns="http://openoffice.org/2007/layout"
+             xmlns:cnt="http://openoffice.org/2007/layout/container"
+             id="RID_SVXDLG_ZOOM" _title="Zoom &amp; View Layout" optimumsize="true"
+	     help-id="SID_ATTR_ZOOM"
+             has_border="true" sizeable="true" moveable="true">
+    <vbox spacing="5" border="5">
+	<hbox spacing="0" border="0">
+	    <vbox spacing="5" border="5">
+		<fixedline id="FL_ZOOM" _text="Zoom factor" cnt:expand="true"/>
+		<radiobutton radiogroup="zoom" id="BTN_OPTIMAL"   _label="Optimal"/>
+		<radiobutton radiogroup="zoom" id="BTN_WHOLE_PAGE" _label="Whole Page"/>
+		<radiobutton radiogroup="zoom" id="BTN_PAGE_WIDTH" _label="Page Width"/>
+		<radiobutton radiogroup="zoom" id="BTN_100"   _label="100 %"/>
+		<hbox cnt:expand="false" cnt:fill="true">
+		    <align cnt:expand="false" cnt:fill="true">
+			<radiobutton cnt:v-align="0.5" cnt:v-fill="0" radiogroup="zoom" id="BTN_USER" _label="Variable"/>
+		    </align>
+		    <flow cnt:padding="10" cnt:expand="false"/>
+		    <metricfield id="ED_USER" value-step="1"
+				 repeat="true" has_border="true" spin="true"
+				 _text="100%" unit="11" custom-unit-text="%"
+				 right="true"
+				 cnt:expand="false"/>
+		</hbox>
+	    </vbox>
+	    <vbox spacing="5" border="5">
+		<fixedline id="FL_VIEWLAYOUT" _text="View layout" cnt:expand="true"/>
+		<radiobutton radiogroup="zoom" id="BTN_AUTOMATIC" _label="~Automatic"/>
+		<radiobutton radiogroup="zoom" id="BTN_SINGLE"    _label="~Single page"/>
+		<hbox cnt:expand="false" cnt:fill="true">
+		    <align cnt:expand="false" cnt:fill="true">
+			<radiobutton cnt:v-align="0.5" cnt:v-fill="0" radiogroup="zoom" id="BTN_COLUMNS" _label="~Columns"/>
+		    </align>
+		    <flow cnt:padding="10" cnt:expand="false"/>
+		    <metricfield id="ED_COLUMNS" value-step="1"
+				 repeat="true" has_border="true" spin="true"
+				 _text="100%"
+				 right="true"
+				 spin-value-min="1"
+				 spin-value="1"
+				 cnt:expand="false"/>
+		</hbox>
+		<hbox><flow cnt:expand="false" cnt:padding="10"/><checkbox id="CHK_BOOK" _label="~Book mode"/></hbox>
+	    </vbox>
+	</hbox>
+	<fixedline cnt:padding="1" id="FL_BOTTOM"/>
+	<dialogbuttonhbox border="5" spacing="5">
+	    <flow/>
+	    <okbutton     id="BTN_ZOOM_OK"/>
+	    <cancelbutton id="BTN_ZOOM_CANCEL"/>
+	    <helpbutton   id="BTN_ZOOM_HELP"/>
+	</dialogbuttonhbox>
+    </vbox>
+</modaldialog>
cvs diff: Diffing svx/util
Index: svx/util/makefile.mk
===================================================================
RCS file: /cvs/graphics/svx/util/makefile.mk,v
retrieving revision 1.56
retrieving revision 1.56.18.2
diff -p -u -u -p -b -w -B -r1.56 -r1.56.18.2
--- svx/util/makefile.mk	12 Mar 2008 10:14:26 -0000	1.56
+++ svx/util/makefile.mk	28 Mar 2008 10:17:48 -0000	1.56.18.2
@@ -188,11 +188,6 @@ SHL2STDLIBS+= \
              $(ADVAPI32LIB)
 .ENDIF # WNT
 
-.IF "$(ENABLE_LAYOUT)" == "TRUE"
-LINKFLAGS+= -L../$(PRJ)/layout/$(INPATH)/lib
-SHL2STDLIBS += -ltklayout$(UPD)$(DLLPOSTFIX)
-.ENDIF # ENABLE_LAYOUT == TRUE
-
 # ------------------------------------------------------------------------------
 
 # Resource files
cvs diff: Diffing svx/util/defs
cvs diff: Diffing svx/workben
cvs diff: Diffing svx/workben/msview
cvs diff: Diffing svx/xml
